imports/topcode-rppt/web/dart/main.dart

```diff
@@ -78,6 +78,19 @@ class RPPT {
 
   RPPT() {
     CanvasElement canvas = querySelector("#video-canvas");
+    print(canvas);
+    ctx = canvas.getContext("2d");
+    scanner = new Scanner();
+    video = querySelector("#video-stream");
+
+    video.autoplay = true;
+    video.onPlay.listen((e) {
+      timer = new Timer.periodic(const Duration(milliseconds : 100), refreshCanvas);
+    });
+  }
+
+  void init() {
```

Comment:
i was sleepy and trying to debug something else -- nothing's actually changed here